recipes/ros-conda-mutex/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: ros-conda-mutex
+  version: 1.0
+
+build:
+  number: 0
+  # need to put this in conda_build_config.yml when more versions of ROS land.
+  string: melodic
+
+about:
+  description: |
+    Mutex package for the ROS infrastructure on conda.
+  license: BSD3
```

Comment:
```suggestion
  license: BSD-3-Clause
```

Reply:
these are not auto generated so I'll use that suggestion :)